plugin/src/test/kotlin/com/exactpro/th2/gradle/BaseTh2PluginTest.kt

```diff
@@ -85,6 +93,45 @@ internal class BaseTh2PluginTest {
         }
     }
 
+    @Test
+    fun `applies license configuration from build script`() {
+        val project =
+            ProjectBuilder.builder().build()
+
+        val customAllowedLicense = "file:test-allowed-licenses.json"
+        val customLicenseNormalizer = Path("src", "test", "resources", "test-license-normalizer-bundle.json").absolutePathString()
```

Comment:
suggestion: Instead of having the useless file in the project sources we could inject a path to a tmp directory and create this file in that directory. The file will be automatically deleted once the test if is completed